_data/members.yml

```diff
@@ -136,11 +136,11 @@
 
 - name:  Sebastian Prillo
   degrees:
-  image: john_d.jpg
+  image: sebastian_prillo.jpg
   altimage:
   position: Graduate student (EECS); joint with Yun Song
-  email:
-  website:
-  scholar:
-  twitter:
+  email: sprillo (at) berkeley.edu
+  website: http://people.eecs.berkeley.edu/~sprillo/
+  scholar: dvJiM_kAAAAJ
```

Comment:
I think this needs the last 3 characters like mine and Nir's and Romain's

Reply:
@adamgayoso really? `&hl` sounds like the language parameter, as in `https://scholar.google.com/citations?hl=en&user=dvJiM_kAAAAJ`. If I use `&hl` as you suggest I get redirected to `https://scholar.google.com/citations?user=dvJiM_kAAAAJ&hl=`. Does this make my google scholar landing page default to english? I'll push the change though, but was just curious to hear why it's needed!

Reply:
oh ok I have no clue honestly haha FOLLOWING PATTERNS